serenata_toolbox/datasets/downloader.py

```diff
@@ -38,7 +38,8 @@ class Downloader:
         '2017-03-15-reimbursements.xz',
         '2017-03-20-purchase-suppliers.xz',
         '2017-04-27-companies-no-geolocation.xz',
```

Comment:
Shouldn't this be removed from this list in light of the new `companies-no-geolocation` dataset?